streams/runners/storm/common/src/main/java/com/hortonworks/streamline/streams/storm/common/StormRestAPIClient.java

```diff
@@ -45,29 +47,29 @@ public StormRestAPIClient(Client client, String stormApiRootUrl, Subject subject
     }
 
     public Map getTopologySummary(String asUser) {
-        return doGetRequest(getTopologySummaryUrl(asUser));
+        return doGetRequest(getTopologySummaryUrl(encode(asUser)));
     }
 
     public Map getTopology(String topologyId, String asUser) {
-        return doGetRequest(getTopologyUrl(topologyId, asUser));
+        return doGetRequest(getTopologyUrl(encode(topologyId), encode(asUser)));
```

Comment:
it will involve refactoring the most of the code since there are other methods like `getComponentUrl` `getTopologySummaryUrl` etc which doesnt follow the above pattern and all parts of the url need not be encoded. For now my preference is to encode in the public methods.

I also moved the encode to WsUtils so it can be used in other places if needed.